crates/router/src/analytics.rs

```diff
@@ -592,7 +592,7 @@ pub mod routes {
                 .await
                 .map(ApplicationResponse::Json)
             },
-            &auth::JWTAuth(Permission::Analytics),
+            &auth::JWTAuth(Permission::RefundRead),
```

Comment:
will users with `RefundWrite` get `RefundRead` permission too?

Reply:
User Roles are made up of groups, and those groups is a set of permissions.

So if there is a group names `OperationsWrite`, then in that group, we put all the read and write permissions for operations like payments, refunds, etc..


So, users will never have only `RefundWrite` permission, if they have it, they will also have `RefundRead` as well. As both of them will be in the same group.